addons/adminmenu/XEH_postServerInit.sqf

```diff
@@ -80,3 +80,13 @@ if (isMultiplayer) then {
         };
     }];
 };
+
+
+if (isNil QGVAR(sideCenter)) then {
+    GVAR(sideCenter) = createCenter sideLogic;
```

Comment:
Is `createCenter` necessary? Looking at the [wiki page](https://community.bistudio.com/wiki/createCenter) it says 

> In Arma 3, centers for all sides are all created by the engine.